doc/lua_api.txt

```diff
@@ -93,25 +93,11 @@ Mods
 
 Mod load path
 -------------
-Generic:
+Paths are relative to the directories listed in the Paths section above.
 
-* `$path_share/games/gameid/mods/`
-* `$path_share/mods/`
-* `$path_user/games/gameid/mods/`
-* `$path_user/mods/` (User-installed mods)
-* `$worldpath/worldmods/`
-
-In a run-in-place version (e.g. the distributed windows version):
-
-* `minetest-0.4.x/games/gameid/mods/`
-* `minetest-0.4.x/mods/` (User-installed mods)
-* `minetest-0.4.x/worlds/worldname/worldmods/`
-
-On an installed version on Linux:
-
-* `/usr/share/minetest/games/gameid/mods/`
-* `$HOME/.minetest/mods/` (User-installed mods)
-* `$HOME/.minetest/worlds/worldname/worldmods`
+* `games/gameid/mods/`
+* `mods/`
+* `worlds/worldname/worldmods/`
```

Comment:
`<gameid>` and `<worldname>` to clarify that they're not static.

Reply:
Now done.